buidl/hash.py

```diff
@@ -0,0 +1,48 @@
+try:
```

Comment:
I like that this is no longer in `helper.py` (kind of a random catch-call that gets too long).

A little confusing that for ecc ops there's `pecc.py` and `cecc.py` and this uses a different layout. Maybe move the below into `pecc.py` (arguably too long and contains lots of unrelated ECC stuff) or do a similar `phash.py` and `chash.py` file setup?

I don't feel strongly, do whatever you like! Also works as-is. Just feedback on readability.